.startup.ipy

```diff
@@ -0,0 +1,48 @@
+>>> from itertools import product
```

Comment:
Committed on purpose?

Reply:
sorry, didn't catch this comment. This was a mistake, was a file i ran on ipython startup related to another PR. It has been removed in the newest version.